templates/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+**Describe what this pull request brings**
+<!--
+Some examples:
+
+1. This pull request brings component Hero for the page Home.
+
+2. This pull request adds fixes for the responsive version of the page Blog.
+
+3. This pull request fixes user assignment via phone number in the existing stay.
+-->
+
+**Describe any additional changes**
+<!--
+An example:
+
+- fixes Header's height
+- introduces a new SASS variables $header-height
+- renames global CSS class "block" into "section"
+- fixes a typo in Hero for Home
+
+If this pull request does not contain any additional changes, feel free to remove this section.
+-->
+
+**Steps to test**
+<!--  Need a good example or template -->
+
+**Screenshots**
+<!-- 
+Attach any relevant screenshots.
+
+If this pull request does not represent any visual changes, feel free to remove this section.
+-->
+
+**Checklist**
```

Comment:
Honestly, I don't know if this section belongs in PR template. I mean, what is the purpose? Let's face it, everybody will be skipping it, mindlessly checking the boxes. Just like with license agreement in apps. 

It is meant to be presented in _every single_ PR, bring value and serve as a signal that PR's author took responsibility, but in reality it will be nothing else but noise.

Reply:
Well, that's just a theory :)

I suggest us to try this **if** we will come up with good points for this checklist